work/test.en.qmd

```diff
@@ -0,0 +1,26 @@
+---
+title: Untitled
+format: html
+---
+
+test of a document in French
+
+shortcode {{< figure src="https://placekitten.com/200/300" caption="A kitten  >}}
+
+$$
+E = m \times c^2
+$$ {#eq-gravity []}
+
+with an online equation $\alpha[{x_{1}}]=1$ which must not be modified
+
+Formally, the gravity model describes the strength of a relationship between two objects as a function of their respective distance and mass. By analogy, the gravity model consists here in evaluating the number of work trips between two locations by taking as the masses the number of inhabitants at the point of departure and the number of jobs at the point of arrival and, as the denominator, a function of f$ increasing with distance[^1]. Thus, if we denote the points of departure by $i$, we have and the end points by $j$ :
```

Comment:
could we shorten this? this way it could be used in actual tests, where we save the API output (so the shorter the text the smaller the fixture file)

Reply:
yes, I was testing one of my texts, I still get a bug somewhere that I have to tack down. I'll simplify everything for a formal test.